contract/views/contract.xml

```diff
@@ -0,0 +1,653 @@
+                                    />
+                                    <field
+                                        name="analytic_tag_ids"
+                                        widget="many2many_tags"
+                                        groups="analytic.group_analytic_tags"
+                                    />
+                                    <field name="quantity" />
+                                    <field name="uom_id" />
+                                    <field
+                                        name="automatic_price"
+                                        attrs="{'column_invisible': [('parent.contract_type', '=', 'purchase')]}"
+                                    />
+                                    <field
+                                        name="price_unit"
+                                        attrs="{'readonly': [('automatic_price', '=', True)]}"
+                                    />
+                                    <field name="specific_price" invisible="1" />
+                                    <field name="discount" groups="base.group_no_one" />
+                                    <field name="price_subtotal" />
+                                    <field name="recurring_interval" invisible="1" />
+                                    <field name="recurring_rule_type" invisible="1" />
+                                    <field
+                                        name="recurring_invoicing_type"
+                                        invisible="1"
+                                    />
+                                    <field name="recurring_next_date" invisible="1" />
+                                    <field name="date_start" invisible="1" />
+                                    <field name="date_end" />
+                                    <field
+                                        name="last_date_invoiced"
+                                        groups="base.group_no_one"
+                                    />
+                                    <field
+                                        name="create_invoice_visibility"
+                                        invisible="1"
+                                    />
+                                    <field
+                                        name="is_plan_successor_allowed"
+                                        invisible="1"
+                                    />
+                                    <field name="is_stop_allowed" invisible="1" />
+                                    <field name="is_cancel_allowed" invisible="1" />
+                                    <field name="is_un_cancel_allowed" invisible="1" />
+                                    <field name="is_canceled" invisible="1" />
+                                    <button
+                                        name="action_plan_successor"
```

Comment:
those actions must be hidden in case `line_recurrency=False`.

We need to have the same actions in contract level

Reply:
I let some actions like stopping a line, as you can do this even if the recurrence is activated. This one was also preserved because although the misleading name, it can be used for resuming a paused line, which is also a possibility with global recurrency.

Reply:
the issue here, is after un-canceling a line or a resume after a stop, the system ask for the next invoice date. The date selected by the user is ignored. This is not user-friendly.

![1](https://user-images.githubusercontent.com/12665409/92224281-e2c61880-eea1-11ea-86c4-97882debb258.png)


I propose to hide all actions in line level if the global mode is selected (only keep end-date editable) and add to the roadmap that we need to adpat those actions to the new mode

Reply:
And what about hiding that field in case of contract recurrence?

Reply:
I still think that it's useful to allow pause/resume even when contract recurrence.

Reply:
it's useful yes, but we need to adapt views to not ask for next invoice date if we are in global mode

Reply:
OK, I'll go that way then, and check the rest of the comments.